src/main/java/uk/gov/companieshouse/exception/GlobalExceptionHandler.java

```diff
@@ -26,44 +30,44 @@
                 .stream()
                 .collect(Collectors.toMap(FieldError::getField, FieldError::getDefaultMessage));
 
-        LOGGER.info("[Unprocessable entity] - {} - {}", request.getRequestURL().toString(), validationErrors);
+        logger.info(String.format("[Unprocessable entity] - %s - %s", request.getRequestURL().toString(), validationErrors));
```

Comment:
Very minor, but you could use the `HttpStatus` enums for these (Unprocessable entity, Unauthorised, etc)

Reply:
Aren't they status codes only rather than text?

Reply:
Ah so it is, I was accidentally looking at the reasonPhrase in the enum rather than the value. Yeah this is fine.